benchmark/benchmark-tidb-using-sysbench.md

```diff
@@ -123,6 +123,10 @@ db-driver=mysql
 
 ### Data import
 
+> **Note:**
+>
+> If you enable the optimistic transaction model (TiDB uses the pessimistic transaction model by default), TiDB rolls back transactions when a concurrency conflict is found. Setting `tidb_disable_txn_auto_retry` to `off` turns on the automatic retry mechanism after meeting a transaction conflict, which can prevent Sysbench from quitting because of the transaction conflict error.
```

Comment:
```suggestion
> If you enable the optimistic transaction model (TiDB uses the pessimistic transaction model by default), TiDB rolls back transactions when a concurrency conflict is found. Setting `tidb_disable_txn_auto_retry` to `off` enables automatic retry upon a transaction conflict, which prevents Sysbench from quitting because of false transaction conflicts.
```

Reply:
事务冲突报错应该是说并没有真的发生冲突，系统误报了冲突。这也是为啥启用自动重试，重试后之前的冲突可能就被证明并不是冲突了。

Reply:
```suggestion
> If you enable the optimistic transaction model (TiDB uses the pessimistic transaction model by default), TiDB rolls back transactions when a concurrency conflict is found. Setting `tidb_disable_txn_auto_retry` to `off` enables the automatic retry mechanism after meeting a transaction conflict, which can prevent Sysbench from quitting because of the transaction conflict error.
```

Reply:
> 这个 PR 只是复制了 4.0 以上 release 同一位置的英文部分。所以我还是将英文保持和其他版本一致哈。